### ISSUE TITLE
[a11y] Cannot visually unhide field

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1313]
**Describe the bug**
In the schema config, cannot unhide a field.

**To Reproduce**
1. Go to the Schema Config
2. Go to the locality table
3. Try to unhide originalElevationUnit
4. See highlighted checkbox, no change.
5. Field is still unhidden

**Screenshots**
![image](https://user-images.githubusercontent.com/37256050/164741952-5099432e-e449-4272-90bd-f544761a03a9.png)

**Desktop (please complete the following information):**
 - OS: Windows 10, Edge, macOS, Edge

**Comment by maxxxxxdlp** [comment]
Fixed in c46d8b21

**Comment by grantfitzsimmons** [comment]
![image](https://user-images.githubusercontent.com/37256050/165801390-49f14229-1077-4431-b08c-def12e860a52.png)

Some fields show a highlight around the element while not displaying as selected. Once saved, it is selected.

**Comment by grantfitzsimmons** [comment]
Same issue as before, except the field format selection is not fixed

**Comment by maxxxxxdlp** [comment]
Fixed in 03b156fa
